test/mesh.jl

```diff
@@ -103,9 +103,9 @@
     @test maximum(sub) == P2(5,8)
 
     # subgrid with comparable vertices of grid
-    grid = CartesianGrid((10,10), P2(0.0,0.0), V2(1.2,1.2))
+    grid = CartesianGrid((10,10), P2(0.0,0.0), SVector{2,T}(1.2,1.2))
```

Comment:
I think this is a good time to review the type of spacing in CartesianGrid. Perhaps it should just be a NTuple to be consistent with the dims field. Would that make the code much uglier? I think all constructors of CartesianGrid should only accept NTuple and Point variations so that end users never need to import StaticArrays here.